frontend/src/Utils/api-discovery/staticModels/apps.ts

```diff
@@ -0,0 +1,113 @@
+  kind: 'APIResourceList',
+  apiVersion: 'v1',
+  groupVersion: 'apps/v1',
+  resources: [
+    {
+      name: 'controllerrevisions',
+      singularName: '',
+      namespaced: true,
+      kind: 'ControllerRevision',
+      verbs: ['create', 'delete', 'deletecollection', 'get', 'list', 'patch', 'update', 'watch'],
+      storageVersionHash: '85nkx63pcBU=',
+    },
+    {
+      name: 'daemonsets',
+      singularName: '',
+      namespaced: true,
+      kind: 'DaemonSet',
+      verbs: ['create', 'delete', 'deletecollection', 'get', 'list', 'patch', 'update', 'watch'],
+      shortNames: ['ds'],
+      categories: ['all'],
+      storageVersionHash: 'dd7pWHUlMKQ=',
+    },
+    {
+      name: 'daemonsets/status',
+      singularName: '',
+      namespaced: true,
+      kind: 'DaemonSet',
+      verbs: ['get', 'patch', 'update'],
+    },
+    {
+      name: 'deployments',
+      singularName: '',
+      namespaced: true,
+      kind: 'Deployment',
+      verbs: ['create', 'delete', 'deletecollection', 'get', 'list', 'patch', 'update', 'watch'],
+      shortNames: ['deploy'],
+      categories: ['all'],
+      storageVersionHash: '8aSe+NMegvE=',
+    },
+    {
+      name: 'deployments/scale',
+      singularName: '',
+      namespaced: true,
+      group: 'autoscaling',
+      version: 'v1',
+      kind: 'Scale',
+      verbs: ['get', 'patch', 'update'],
+    },
+    {
+      name: 'deployments/status',
+      singularName: '',
+      namespaced: true,
+      kind: 'Deployment',
+      verbs: ['get', 'patch', 'update'],
+    },
+    {
+      name: 'replicasets',
+      singularName: '',
+      namespaced: true,
+      kind: 'ReplicaSet',
+      verbs: ['create', 'delete', 'deletecollection', 'get', 'list', 'patch', 'update', 'watch'],
+      shortNames: ['rs'],
+      categories: ['all'],
+      storageVersionHash: 'P1RzHs8/mWQ=',
+    },
+    {
+      name: 'replicasets/scale',
+      singularName: '',
+      namespaced: true,
+      group: 'autoscaling',
+      version: 'v1',
+      kind: 'Scale',
+      verbs: ['get', 'patch', 'update'],
+    },
+    {
+      name: 'replicasets/status',
+      singularName: '',
+      namespaced: true,
+      kind: 'ReplicaSet',
+      verbs: ['get', 'patch', 'update'],
+    },
+    {
+      name: 'statefulsets',
+      singularName: '',
+      namespaced: true,
+      kind: 'StatefulSet',
+      verbs: ['create', 'delete', 'deletecollection', 'get', 'list', 'patch', 'update', 'watch'],
+      shortNames: ['sts'],
+      categories: ['all'],
+      storageVersionHash: 'H+vl74LkKdo=',
+    },
+    {
+      name: 'statefulsets/scale',
+      singularName: '',
+      namespaced: true,
+      group: 'autoscaling',
+      version: 'v1',
+      kind: 'Scale',
+      verbs: ['get', 'patch', 'update'],
+    },
+    {
+      name: 'statefulsets/status',
+      singularName: '',
+      namespaced: true,
+      kind: 'StatefulSet',
+      verbs: ['get', 'patch', 'update'],
+    },
+  ],
+} as APIResourceList;
```

Comment:
@karelhala I know it's probably easier to use type assertions here, but could we update to use type annotations instead to avoid weakening our type checking?

Reply:
Good point!